### ISSUE TITLE
Marker popups do not include directions after applying filter or searching

### ISSUE THREAD

**Issue by talllguy** [kcigeospatial/tmdl-grass-swale-map#30]
When the user filters or applies a search to the markers, the first popup will not contain the interactive directions specified in the function [here](https://github.com/kcigeospatial/tmdl-grass-swale-map/blob/master/index.html#L116-L131). I added a `click` event trigger to the `featurelayer.on` AJAX call to try and fix this, since `ready` only occurs once. However, the `click` event is merely a workaround, and it doesn't work the first time.

Is there a way to make this function run every time, rather than only after `ready` or `click`.

**Comment by taojun545** [comment]
Fixed the issue on new PR.

http://stackoverflow.com/questions/22716647/geojson-marker-from-url-and-setfilter-then-no-more-custom-popup
